docs/settings.rst

```diff
@@ -69,3 +69,28 @@ exceeds 90% or available memory drops below 100 MB.
 
    Specify the desired memory utilization threshold, in megabytes. When available
    memory falls below the specified value, a warning will be reported.
+
+Celery Health Check
+----------------------
+Using `django.settings` you may exert more fine-grained control over the behavior of the celery health check
+
+.. list-table:: Additional Settings
+   :widths: 25 10 10 55
+   :header-rows: 1
+
+   * - Name
+     - Type
+     - Default
+     - Description
+   * - `HEALTHCHECK_CELERY_TIMEOUT`
```

Comment:
You don't need to document `HEALTHCHECK_CELERY_TIMEOUT` since it deprecated. Only document the behavior for the new settings. You can also ignore the fallback in the documentation.

Reply:
🎉 updated